crates/router/src/connector/bitpay/transformers.rs

```diff
@@ -279,6 +280,7 @@ fn get_crypto_specific_payment_data(
         ConnectorAuthType::HeaderKey { api_key } => api_key,
         _ => String::default().into(),
     };
+    let order_id = item.connector_request_reference_id.clone();
```

Comment:
```suggestion
    let order_id = item.router_data.connector_request_reference_id.clone();
```

Reply:
@Himanshu-370 Please do the above change

Reply:
done